Sources/Orbit/Components/ListChoice.swift

```diff
@@ -163,19 +174,21 @@ public extension ListChoice {
     init(
         _ title: String = "",
         description: String = "",
-        icon: Icon.Content,
+        iconContent: Icon.Content,
         disclosure: ListChoiceDisclosure = .disclosure(),
         showSeparator: Bool = true,
         action: @escaping () -> Void = {},
-        @ViewBuilder content: @escaping () -> Content
+        @ViewBuilder content: @escaping () -> Content,
+        @ViewBuilder headerContent: @escaping () -> HeaderContent
```

Comment:
Should `content` precede `headerContent`? I'd expect it to be the other way around...

Reply:
It is based on the later SwiftUI, where API has been changed in favor of putting the "more important" closure first, so that it can be used as a first, unnamed, closure. The API change can be seen for example in `SwiftUI.Section` component.

Deprecated:
<img width="1074" alt="image" src="https://user-images.githubusercontent.com/35844477/170270049-76be7853-46e0-442e-a12b-abaf43dfa9c1.png">

New:
<img width="1112" alt="image" src="https://user-images.githubusercontent.com/35844477/170269989-7fc4d723-1b2b-4e43-b2be-c8a136bd17e2.png">